src/renderer/prefComponents/editor/index.vue

```diff
@@ -34,20 +34,20 @@
       :onChange="value => onSelectChange('codeFontSize', value)"
     ></range>
     <font-text-box
-      description="The font used in code blocks."
+      description="Font family in code blocks."
       :onlyMonospace="true"
       :value="codeFontFamily"
       :onChange="value => onSelectChange('codeFontFamily', value)"
     ></font-text-box>
     <!-- FIXME: Disabled due to #1648. -->
     <bool
       v-show="false"
-      description="Whether to show line numbers in code blocks."
+      description="Show line numbers in code blocks."
       :bool="codeBlockLineNumbers"
       :onChange="value => onSelectChange('codeBlockLineNumbers', value)"
     ></bool>
     <bool
-      description="Trim any empty lines at the beginning and end of code blocks when you open a markdown file."
+      description="Handling of leading and trailing empty lines."
```

Comment:
Since this is a checkbox, I feel that there should be some kind of explanation of what this actually does, something like this?

```suggestion
      description="Whether to trim empty lines at the beginning and end of code blocks upon opening a file."
```

Reply:
Is this text in use? I thought it corresponded the options list. If it is a checkbox, then a change is needed, but the "Whether to" opening is unnecessarily verbose.

Reply:
The old string suggests to me that this is a checkbox, @Jocs is it?

Reply:
A checkbox label is normally a verb. I would support "Trim empty lines at the beginning and end of code blocks upon opening a file."

Reply:
White space includes spaces and tabs, which are obviously critical in code. It seems a little better to clarify this distinction for the user. Perhaps "empty lines" instead?

Reply:
My bad, I've checked that and we only remove empty lines at the beginning and end of the code block.

Reply:
So?:

- "[Remove|Trim] empty lines at [start|beginning] and end of code blocks"
- "[Remove|Trim] leading and trailing empty lines in code blocks"

Reply:
I'm fine with both remove and trim but think the first one fits better in the context. 
```suggestion
      description="Remove leading and trailing empty lines in code blocks."
```